lib/code_corps/github/user.ex

```diff
@@ -0,0 +1,25 @@
+defmodule CodeCorps.GitHub.User do
```

Comment:
This could be made into a struct, for a bit more "typechecking", using `defstruct` or even using an ecto embedded schema.